fields/uabb-gradient/uabb-gradient.php

```diff
@@ -1,107 +1,113 @@
-<?php 
-/*
-	Declaration of array
```

Comment:
show this declaration in the doc comments of the  uabb-gradient function

Reply:
Added